src/main/java/edu/cornell/kfs/module/purap/service/impl/CuPurapGeneralLedgerServiceImpl.java

```diff
@@ -72,136 +72,144 @@ protected int getNextAvailableSequence(final String documentNumber) {
         return count + 1;
     }
     
-    protected boolean generateEntriesPaymentRequest(
```

Comment:
The reason it was fully commented out was to get the code to compile. Multiple issues popped up between base code and our customization. Thanks for catching that it was still commented out.

I have had to jump through some hoops to retain our customization and its not pretty. I took the approach I did because of where in the if-clause our customization needs to occur. If you think the extra comments don't add value let me know.